java-test-samples/apigw-lambda-ddb/LoadTests/load-test-static-data.yml

```diff
@@ -0,0 +1,20 @@
+config:
+  ## Target will be your api endpoint base url
+  target: "<<APIGW ENDPOINT>>/<<STAGE>>"
+  phases:
+    - duration: 10
```

Comment:
Ideally, we should run it little bit higher duration (say, 60) with a higher arrival rate (say, 100)

Reply:
Tested with higher load but changed to lower values while doing more tests. Will change it back to bigger values.

Reply:
done